### PR TITLE
[OSS-1] Disable adding duplicated tags and fix for angular 1.3+

### PR DESCRIPTION
**Pull request by wojtekmaciej**
https://monterail.atlassian.net/browse/OSS-1

[Review on Reviewable](https://reviewable.io/reviews/monterail/angular-tagger/16)

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
     "build/angular-tagger.css"
   ],
   "dependencies": {
-    "angular": "~1.0.7"
+    "angular": "~1.4.8"
```
Comment (jandudulski) [inline bower.json]: Are you sure? Do we really need to set such strict dependency?

File: src/angular-tagger.coffee [hunk 2]
```diff
@@ -261,8 +262,7 @@ angular.module("tagger").directive "tagger", ["$compile", "$timeout", ($compile,
         $scope.config.createFun($scope.query)
       else if $scope.selected > -1
         $scope.matching[$scope.selected]
-
-      if item
+      if($scope.tags.indexOf(item) == -1)
         console.log "adding ", item
```
Comment (jandudulski) [inline src/angular-tagger.coffee]: Please also remove the `console.log`

**Comment by wojtekmaciej** [comment]
Review status: 0 of 4 files reviewed at latest revision, 2 unresolved discussions.

---

*[bower.json, line 15 \[r1\]](https://reviewable.io:443/reviews/monterail/angular-tagger/16#-K6DKKO9zbHowKSb3IE0-r1-15:-K6DKR41OR9kLioacEdk:-1856938562) ([raw file](https://github.com/monterail/angular-tagger/blob/30e85560b13bc4b80bb4e51ae678d354ea740335/bower.json#L15)):*
Of course no, I checked how it works with new version of angular

---

*[src/angular-tagger.coffee, line 266 \[r1\]](https://reviewable.io:443/reviews/monterail/angular-tagger/16#-K6DKKO9zbHowKSb3IE3-r1-266:-K6DKQA8xWvP0VXEcUli:-499146009) ([raw file](https://github.com/monterail/angular-tagger/blob/30e85560b13bc4b80bb4e51ae678d354ea740335/src/angular-tagger.coffee#L266)):*
Done.

---


*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/monterail/angular-tagger/16)*

**Comment by kubakrzempek** [comment]
Looks cool to me, @jandudulski can you acknowledge the changes, please?

---

Reviewed 2 of 4 files at r1, 2 of 2 files at r2.
Review status: all files reviewed at latest revision, 2 unresolved discussions.

---



*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/monterail/angular-tagger/16#-:-KAEUqNIYPyay0vD6jq4:1022589849)*

**Pull request merged**